http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -151,9 +153,22 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
       app shouldBe defined
       app.get.app.status shouldBe AppStatus.Running
       app.get.cluster.asyncFields shouldBe defined
+      app
     }
 
-    res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+    val dbApp = res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+    dbApp shouldBe defined
+    val app = dbApp.get.app
+
+    for {
+      _ <- aksInterp.deleteApp(DeleteAKSAppParams(app.appName, workspaceId, cloudContext))
+      app <- KubernetesServiceDbQueries
+        .getActiveFullAppByName(CloudContext.Azure(cloudContext), app.appName)
+        .transaction
+    } yield {
+      app shouldBe defined
+      app.get.app.status shouldBe AppStatus.Deleted
+    }
```

Comment:
This seems like enough -- Sam/helm/etc are all mocked out but we can assert DB status.

I don't think this last for-comp is running though because there is no `unsafeRunSync`..